### PR TITLE
3mdeb at Embedded World 2019

### PR DESCRIPTION
**Pull request by MartaSzel**

### CODE REVIEW

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+categories:
+    - Miscellaneous
+---
+
+##3mdeb at Embedded World 2019##
+
+![ew2019-logo](link)
+
+We are excited to inform You that 3mdeb will take part in embedded world
+Exhibition & Conference - [**Embedded World 2019**](https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-43765140/3mdeb-embedded-systems-consulting-piotr-krol) next week! In place full of enthusiasts You can meet us personally between 26.02-28.02 in Nuremberg, Germany. Come and talk with us about amazing world of embedded systems. We will tell You
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: wrap to 80 characters (or new line after long url)

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+not only about our work and innovations, but also share our knowledge, help and
+present to You our demos:
+* [Remote Testing Environment](https://cloud.3mdeb.com/index.php/s/Jt6PEtos7iznQkf) - Hat designed for Orange Pi Zeroboard which runs
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: Zero board

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - RTE
+    - TPM2.0
+    - MuxPi
+    - BayTrail
+
+categories:
+    - Miscellaneous
+---
+
+##3mdeb at Embedded World 2019##
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: ## 3mdeb at Embedded World 2019

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 4]
```diff
@@ -18,33 +18,61 @@ categories:
     - Miscellaneous
 ---
 
-##3mdeb at Embedded World 2019##
+## 3mdeb at Embedded World 2019
 
 ![ew2019-logo](link)
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: https://3mdeb.com/wp-content/uploads/2019/02/ew2019-logo.jpg

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 5]
```diff
@@ -18,33 +18,61 @@ categories:
...
+coreboot - in comparison to UEFI implementation on the example
+of BayTrail platform - MinnowBoard Turbot. Additionally, we
+present how fast can embedded platform boot with open source
+firmware compared to official firmware. The platforms are running
+the same operating system (in UEFI and legacy mode) in multi-user
+target. The chosen distro is lightweight Solus OS. Using common
+Linux and coreboot tools, boot times are presented of both
+firmware implementations.
 
 ![flyers](link)
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: https://3mdeb.com/wp-content/uploads/2019/02/ew2019-fliers.png

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 6]
```diff
@@ -18,33 +18,61 @@ categories:
...
+You can see the samples of what we have to present here:
+* [What will happen if your hypervisor ROT13 whole Ubuntu VM output](https://asciinema.org/a/228858)
+* [Embedded World 2019 - Bareflank demo ](https://asciinema.org/a/228849)
 
-Do you feel interested? Then… Can’t wait to see You there! Look for us in Hall 3 / 3-563.
+Do you feel interested? Then… You can leave us a message or desired Your appointment
+during the exhibition [HERE](https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-43765140/3mdeb-embedded-systems-consulting-piotr-krol#exhibitorcontact).
+Can’t wait to see You there! Look for us in Hall 3 / 3-563.
 
 ![map](link)
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: https://3mdeb.com/wp-content/uploads/2019/02/ew2019-booth.png

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 7]
```diff
@@ -18,33 +18,61 @@ categories:
...
 We are excited to inform You that 3mdeb will take part in embedded world
-Exhibition & Conference - [**Embedded World 2019**](https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-43765140/3mdeb-embedded-systems-consulting-piotr-krol) next week! In place full of enthusiasts You can meet us personally between 26.02-28.02 in Nuremberg, Germany. Come and talk with us about amazing world of embedded systems. We will tell You
-not only about our work and innovations, but also share our knowledge, help and
-present to You our demos:
-* [Remote Testing Environment](https://cloud.3mdeb.com/index.php/s/Jt6PEtos7iznQkf) - Hat designed for Orange Pi Zeroboard which runs
+Exhibition & Conference - [**Embedded World 2019**](https://www.embedded-world.de/en/ausstellerprodukte/embwld19/exhibitor-43765140/3mdeb-embedded-systems-consulting-piotr-krol)
+next week! In place full of enthusiasts You can meet us personally between
+26.02-28.02 in Nuremberg, Germany. Come and talk with us about amazing world of
+embedded systems. We will tell You not only about our work and innovations, but
+also share our knowledge, help and present to You our demos:
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: our products and demos:

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 8]
```diff
@@ -18,33 +18,61 @@ categories:
...
+* [Remote Testing Environment](https://cloud.3mdeb.com/index.php/s/Jt6PEtos7iznQkf) -
+Hat designed for Orange Pi Zero board which runs
 specially craftedLinux distribution using the YoctoProject
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: crafted Linux

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 9]
```diff
@@ -18,33 +18,61 @@ categories:
...
-Minnowboard Turbot.
+* [TPM2.0](https://cloud.3mdeb.com/index.php/s/jKgCHCtTSQYejTa) -
+a Trusted Platform Module is a specialized chip on an endpoint device that stores
+RSA encryption keys specific to the host system for hardware
+authentication.
+* [MuxPi](https://cloud.3mdeb.com/index.php/s/WgpRzGYitfEYfBN) -
+a product mainly for developers and testers with the
+need of automating everyday tasks. It comes with help
+providing ease of use, distribution and connectivity.
+* [BayTrailVerified Boot](https://cloud.3mdeb.com/index.php/s/E43E8rinTiYLbG3) -
```
Comment (artur-rs) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: this demo will not be shown

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+    - MuxPi
+    - BayTrail
+
+categories:
+    - Miscellaneous
+---
+
+## 3mdeb at Embedded World 2019
+
+![ew2019-logo](https://3mdeb.com/wp-content/uploads/2019/02/ew2019-logo.jpg)
```
Comment (pietrushnic) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: @artur-rs maybe we should change that graphics with this: https://cloud.3mdeb.com/index.php/s/94Cdq3AG8CdjqgH

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+specially crafted Linux distribution using the Yocto Project.
+* [TPM2.0](https://cloud.3mdeb.com/index.php/s/jKgCHCtTSQYejTa) -
+a Trusted Platform Module is a specialized chip on an endpoint device that stores
+RSA encryption keys specific to the host system for hardware
+authentication.
+* [MuxPi](https://cloud.3mdeb.com/index.php/s/WgpRzGYitfEYfBN) -
+a product mainly for developers and testers with the
+need of automating everyday tasks. It comes with help
+providing ease of use, distribution and connectivity.
+* [BayTrailFast Boot](https://cloud.3mdeb.com/index.php/s/n2A3268kssiHJ96) -
```
Comment (pietrushnic) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: @artur-rs Bay Trail FastBoot

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+the demo is presenting alternative open source firmware -
+coreboot - in comparison to UEFI implementation on the example
+of BayTrail platform - MinnowBoard Turbot. Additionally, we
+present how fast can embedded platform boot with open source
+firmware compared to official firmware. The platforms are running
+the same operating system (in UEFI and legacy mode) in multi-user
+target. The chosen distro is lightweight Solus OS. Using common
+Linux and coreboot tools, boot times are presented of both
+firmware implementations.
+* [BayTrailHypervisor](https://cloud.3mdeb.com/index.php/s/QZPnMfXTsipQeES) -
```
Comment (pietrushnic) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: @artur-rs Bare Metal Hypervisor?

File: _posts/2019-02-22-3mdeb-at-embedded-world-2019.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+the demo is presenting one of the features that bare metal
+hypervisor gives. It is run in edk2 UEFI implementation on
+MinnowBoard Turbot platform. Hypervisor was written with
+Bareflank - an open source hypervisor SDK. Depending on the
+implementation, it can provide an interface for underlying software
+to control hypervisor's operation or be completely transparent. This
+kind of hypervisors use Intel's VMX (Virtual Machine eXtensions).
+
```
Comment (pietrushnic) [inline _posts/2019-02-22-3mdeb-at-embedded-world-2019.md]: @artur-rs redundant line

**Comment by artur-rs** [comment]
@pietrushnic all conversation should be resolved, the blog post has been published already

**Pull request merged**